packages/vite-app-ts/src/MainPage.tsx

```diff
@@ -99,44 +97,47 @@ export const Main: FC = () => {
     setRoute(window.location.pathname);
   }, [setRoute]);
 
+  // -----------------------------
+  // 📃 Page List
+  // -----------------------------
+  // This is the list of pages and tabs
+  const pageList: TContractPageList = {
+    mainPage: {
```

Comment:
Not a huge deal, but could also add the new YourToken contract here

Reply:
added!